### DISCUSSION TITLE
What dose the model SYM+ASC+R10 mean?

### DISCUSSION THREAD

**Discussion by Fisher909** [iqtree/iqtree2#238] [Q&A] [answered]
What dose the model SYM+ASC+R10 mean?

**Comment by roblanf** [comment, marked as answer]
* SYM: "Symmetric model with unequal rates but equal base freq."
* ASC: Ascertainment Bias Correction (for when you don't have the constant sites in your alignment)
* R10: free-rate distribution with 10 parameters

All of these are covered in the documentation here: http://www.iqtree.org/doc/Substitution-Models
